rules/st2workroom_st2installer_st2enterprise_test_master_ubuntu14.yaml

```diff
@@ -0,0 +1,31 @@
+---
```

Comment:
For now I'm also fine with skipping those tests in the release pipeline.

Just testing on master should be a good start.